soccer/gameplay/tactics/coordinated_pass.py

```diff
@@ -32,10 +32,16 @@ class State(enum.Enum):
         receiving = 3   # the kicker has kicked and the receiver is trying to get the ball
 
 
-    def __init__(self, receive_point=None):
+    def __init__(self, receive_point=None, skillreceiver=None):
```

Comment:
Let's make a note here or at the top about what the skillreceiver is used for / why it's a parameter.